pkg/config/protocol/instance.go

```diff
@@ -63,6 +66,8 @@ func Parse(s string) Instance {
 		return GRPCWeb
 	case "http":
 		return HTTP
+	case "http_proxy":
+		return HTTP_PROXY
```

Comment:
shouldnt the is HTTP functions be updated as well ?

Reply:
Updated isHttp function. Not sure if we have to update other - I do not think so. But please let me know if I missed any thing.